README.md

```diff
@@ -100,7 +100,7 @@ gs.set_cookie_auto('chrome') # search specific browser
 ## submodules
 ### wishes
 Gets your wish history.
-For this you must first open the history/details page in genshin impact,
+For this you **have to open first the history/details page** in genshin impact,
```

Comment:
any reason to change this too?

Reply:
Nothing special here, just rephrasing and highlighting

Reply:
Seems unrelated to everything else but sure I suppose.